pkg/reconciler/openshift/common/transformer.go

```diff
@@ -84,3 +84,34 @@ func RemoveRunAsGroup() mf.Transformer {
 		return nil
 	}
 }
+
+// RemoveFsGroup will remove FsGroup in a deployment
+func RemoveFsGroup() mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		if u.GetKind() != "Deployment" {
+			return nil
+		}
+
+		d := &appsv1.Deployment{}
+		err := runtime.DefaultUnstructuredConverter.FromUnstructured(u.Object, d)
+		if err != nil {
+			return err
+		}
+
+		if d.Name == "tekton-hub-api" {
```

Comment:
pass this as a param to the RemoveFsGroup

Reply:
:P 
https://www.reddit.com/r/ProgrammerHumor/comments/4o0cyr/it_should_be_noted_that_no_ethicallytrained/